src/articles/articles.controller.ts

```diff
@@ -23,22 +29,37 @@ export class ArticlesController {
   ) {}
 
   @Post('create')
```

Comment:
According to REST, better to not add the action as part of the URL, if it could be express via the HTTP method. 
In this case better to use just `@Post()`